### PR TITLE
Introduce GitHub actions

### PR DESCRIPTION
**Pull request by clem21**

### CODE REVIEW

**Review by pamil — CHANGES_REQUESTED** [review]
The build is failing: https://github.com/clem21/ShopApiPlugin/actions

**Comment by lchrusciel** [comment]
This can be finished after the merge of https://github.com/Sylius/ShopApiPlugin/pull/688 and release of 1.2

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -40,7 +40,11 @@ jobs:
                 uses: shivammathur/setup-php@v2
                 with:
                     php-version: "${{ matrix.php }}"
+<<<<<<< HEAD
```
Comment (mamazu) [inline .github/workflows/build.yml:43]: You seem to commit a merge conflict.

File: tests/Controller/Cart/SummarizeApiTest.php [hunk 2]
```diff
@@ -199,7 +199,7 @@ public function it_shows_summary_of_a_cart_with_coupon_applied(): void
         $bus->dispatch(new PutSimpleItemToCart($token, 'LOGAN_MUG_CODE', 5));
         $bus->dispatch(new AddCoupon($token, 'BANANAS'));
 
-        $this->client->request('GET', '/shop-api/checkout/' . $token, [], [], self::CONTENT_TYPE_HEADER);
+        $this->client->request('GET', '/shop-api/carts/' . $token, [], [], self::CONTENT_TYPE_HEADER);
```
Comment (mamazu) [inline tests/Controller/Cart/SummarizeApiTest.php:202]: Why did that change? Didn't that work before?

**Review by mamazu — CHANGES_REQUESTED** [review]

**Comment by lchrusciel** [comment]
Thanks, Clemence! 🎉

**Pull request merged**